webassembly/README.md

```diff
@@ -0,0 +1,4 @@
+F3D is a fast and minimalist 3D viewer. Implemented in C++, it has been cross-compiled with emscripten to a WebAssembly module.
```

Comment:
This readme file is the one that will be displayed here: https://www.npmjs.com/package/f3d

Reply:
nice

Reply:
I dont see anything to publish the npm, i suppose it comes in another PR ?

Reply:
Yes indeed. Publishing will come later.  
It's possible to do it manually with `npm publish` though.

Reply:
noted